test/DynamoCoreTests/DynamoViewModelUnitTest.cs

```diff
@@ -222,4 +222,4 @@ protected RuntimeMirror GetRuntimeMirror(string varName)
         }
```

Comment:
Please do a `git checkout master test/DynamoCoreTests/DynamoViewModelUnitTest.cs` do revert this file, it is unnecessarily polluting the PR.